pgmigrate: add warnings when large objects present

mble opened the pull request:
### Proposed changes in this pull request

Large objects are not compatible with logical replication as of writing
(https://www.postgresql.org/docs/15/logical-replication-restrictions.html).

Add some warning output if we detect the presence of large objects, and
also update our docs to indicate this.

### Type (put an `x` where ever applicable)
- [ ] Bug fix: Link to the issue
- [X] Feature (Non-breaking change)
- [ ] Feature (Breaking change)
- [X] Documentation Improvement
- [ ] Other

### Checklist
Please put an `x` against the checkboxes. Write a small comment explaining if its `N/A` (not applicable)

- [X] All the tests are passing after the introduction of new changes.
- [ ] Added tests respective to the part of code I have written.
- [X] Added proper documentation where ever applicable (in code and README.md).

### Optional extra information

mble commented:
Related to https://github.com/aiven/aiven-db-migrate/issues/34

kmichel-aiven reviewed (changes requested):
The feature is sound but can you add some tests for it and split the commits with unrelated README/.gitignore changes ?

The new warning in the README is related but it's not clear why all the newlines and the list bullets need to be changed.

mble commented:
@kmichel-aiven Sure. The newlines etc in README come from my auto-formatter for markdown files. I'll disable it and add the warning.

Commit:
gitignore: add vscode and venv to gitignore

Commit:
doc: add logical replication warning to README

Commit:
pgmigrate: add warnings when large objects present

Large objects are not compatible with logical replication as of writing
(https://www.postgresql.org/docs/15/logical-replication-restrictions.html).

Add some warning output if we detect the presence of large objects, and
also update our docs to indicate this.

mble commented:
@kmichel-aiven changes requested have been made.
